tests/v2/validation/rbac/rbac_psa_test.go

```diff
@@ -45,7 +47,7 @@ type PSATestSuite struct {
 
 func (rb *PSATestSuite) TearDownSuite() {
 	// reset the PSACT
-	_, err := editPsactCluster(rb.client, rb.clusterName, defaultNamespace, "rancher-privileged")
+	_, err := editPsactCluster(rb.client, rb.clusterName, defaultNamespace, "")
```

Comment:
wouldn't this be something like `rancher-baseline`? or do we allow an empty policy?

Reply:
we do, but when we run this test its with rancher-privileged set beforehand. So essentially back to its initial state.

Reply:
This is actually fine as is. Israel and I discussed offline, this is how the RBAC test needs to run.